lib/features/steps/session_tracking_steps.rb

```diff
@@ -113,3 +113,83 @@
     }
   end
 end
+
+# Tests that a session payload element is true.
+#
+# @step_input field_path [String] Path to the tested element
+Then('the session payload field {string} is true') do |field_path|
```

Comment:
There are now a lot of these near-duplicate steps for errors & sessions. Is it worth implementing both of these in terms of a generic step so that all of the logic is shared? e.g.

```ruby
Then('the {string} payload field {string} is true') do |request_type, field_path|
  body = Server.send(request_type).current[:body]

  assert_equal(true, read_key_path(body, field_path)) 
end
```

It might be worth making a new method on `Server` for this, e.g.

```ruby
class Server
  def self.type(type)
    case type
    when 'error'
    when 'errors'
      self.errors
    when 'session'
    when 'sessions'
      self.sessions
    else
      raise "Invalid request type '#{type}'"
  end
end
```

Reply:
Yes, definitely.  I wasn't at all happy about how I left it, but I wasn't sure where it was going to end up and what the right level of genericism would look like.  I think the dust has settled now though, so I'll raise a ticket to action this comment specifically before looking to release.  There is another PR waiting in the wings that's based on this, so I I don't want to unsettle too much during the review.

Reply:
Internal ticket PLAT-5661.